python/nav/jwtconf.py

```diff
@@ -3,11 +3,15 @@
 from functools import partial
 import configparser
 from typing import Any
+from datetime import timedelta
 
 from nav.config import ConfigurationError, NAVConfigParser
 
 _logger = logging.getLogger('nav.jwtconf')
 
+ACCESS_TOKEN_EXPIRE_DELTA = timedelta(hours=1)
+REFRESH_TOKEN_EXPIRE_DELTA = timedelta(days=1)
```

Comment:
I'm thinking these would do best as configurable options - but this could easily be worked in as a separate PR.

Make an issue so you don't forget :)

Reply:
Yeah that was the plan

Reply:
#3016